renovate.json

```diff
@@ -5,5 +5,19 @@
   "ignoreDeps": [
     "chalk"
   ],
-  "ignorePaths": ["**/node_modules/**"]
+  "ignorePaths": [
+    "**/node_modules/**"
+  ],
+  "packageRules": [
+    {
+      "enabled": false,
+      "groupName": "ignored js dependencies",
+      "matchManagers": [
+        "npm"
+      ],
+      "matchPackageNames": [
+        "@storybook/builder-vite"
+      ]
+    }
+  ]
```

Comment:
I did not catch we already pinned a dependency. So I think we could just append `@storybook/builder-vite` to `ìgnoreDeps` array